ludwig/collect.py

```diff
@@ -172,7 +172,7 @@ def print_model_summary(model_path: str, **kwargs) -> None:
     """
     model = LudwigModel.load(model_path)
     # Model's dict inputs are wrapped in a list, required by torchinfo.
-    torchinfo.summary(model.model, input_data=[model.model.get_model_inputs()])
+    print(torchinfo.summary(model.model, input_data=[model.model.get_model_inputs()]))
```

Comment:
Is it worth making this a `logger.info()`?

Reply:
Done.